.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ attach-to-workspace: &attach-to-workspace
     at: .
 
 system-builder-ruby25: &system-builder-ruby25
-  image: quay.io/3scale/system-builder:ruby25
+  image: quay.io/3scale/system-builder:node10
```

Comment:
Just make a commit in this PR https://github.com/3scale/system-builder/pull/15

Reply:
Can you make a PR on top of the ruby-25 branch then? And once merge in system-builder no need to change Porta code